### DISCUSSION TITLE
Welcome to LELEC210X Discussions!

### DISCUSSION THREAD

**Discussion by jeertmans** [LELEC210X/LELEC210X#1] [Announcements]
## 👋 Welcome!
 
We’re using Discussions as a place to exchange between students and the teaching staff.

Here, you can:
  * Ask questions you’re wondering about.
  * Share ideas about the project.
  * Propose enhancements in the repository.

**Comment by Matabs05** [comment]
Hello,
I have a problem with the jupyter notebook. After installing rye and following the steps noted in the README, when I want to run the 3rd cell of the notebook it returns me this error. I don't see how to resolve it.
![erreur1](https://github.com/user-attachments/assets/f6c71be7-f06c-4be0-a155-62f08c68b43d)
![erreur2](https://github.com/user-attachments/assets/679c2a6c-2b92-488c-9073-2e31b9d436fb)

**Reply by jeertmans** [reply]
Hello @Matabs05! This looks like an error with PortAudio, which looks similar to #27: did you check that issue? I linked to a potential fix.

Also, please don't use this thread to ask for help, but create another discussion / issue next time :-)

**Reply by jeertmans** [reply]
I am locking this discussion @Matabs05, please comment on #27 if your issue is similar, or create another issue :-)
